app/connectors/aft/AFTConnector.scala

```diff
@@ -82,11 +88,11 @@ class AFTConnectorImpl @Inject()(http: HttpClient, config: FrontendAppConfig) ex
     }
   }
 
-  def endDate: LocalDate = Quarters.getCurrentQuarter.endDate
+  def aftEndDate: LocalDate = Quarters.getCurrentQuarter.endDate
```

Comment:
it should have override modifier if it is overriden

Reply:
added override